frontend/src/pages/Admin/Contacts/index.tsx

```diff
@@ -97,6 +97,7 @@ export const Contacts: FC = () => {
     { width: "half", title: "Postal Code", value: contactQuery?.data?.postal },
     { width: "half", title: "Mobile Phone", value: contactQuery?.data?.mobile },
     { width: "half", title: "Website", value: contactQuery?.data?.website },
+    { width: "half", title: "Email", value: contactQuery?.data?.email },
```

Comment:
@ASpiteri-BCGov  fyi this change in the frontend

Reply:
Yeah sorry I just noticed while I was testing contacts that one field was missing on the read form and didn't want to leave it.